File: testdata/changeSignature/fromScala/AddWithDefault_after.java
public class SimpleMethodJava extends SimpleMethodScala {
  @Override
  public void foo(int i, Object s, boolean b) {
    super.foo(i, foo$default$2(), true);
  }

  public void test() {
    foo(1, foo$default$2(), true);
    SimpleMethodJava smj = new SimpleMethodJava();
    smj.foo(1, smj.foo$default$2(), true);
  }
}

File: testdata/changeSignature/fromScala/AddWithDefault_after.scala
class SimpleMethodScala {
  def foo(i: Int, s: AnyRef = "hi", b: Boolean): Unit = {
    i
  }
}

class SimpleMethodChild extends SimpleMethodScala {
  override def foo(i: Int, s: AnyRef, b: Boolean): Unit = {
    i
    super.foo(i, b = true)
  }
}